apps/prairielearn/src/pages/publicInstructorAssessmentQuestions/publicInstructorAssessmentQuestions.sql

```diff
@@ -0,0 +1,99 @@
+-- BLOCK select_assessment_by_id
+SELECT
+  *
+FROM
+  assessments
+where
+  id = $assessment_id;
+
+-- BLOCK check_course_instance_is_public
+SELECT
+  ci.shared_publicly
+FROM
+  course_instances AS ci
+WHERE
+  ci.id = $course_instance_id;
+
+
+-- BLOCK questions
+WITH
+  issue_count AS (
+    SELECT
+      q.id AS question_id,
+      count(*) AS open_issue_count
+    FROM
+      issues AS i
+      JOIN questions AS q ON (q.id = i.question_id)
+    WHERE
+      i.assessment_id = $assessment_id
+      AND i.course_caused
+      AND i.open
+    GROUP BY
+      q.id
+  )
+SELECT
+  aq.*,
+  q.qid,
+  q.title,
+  row_to_json(top) AS topic,
+  q.id AS question_id,
+  admin_assessment_question_number (aq.id) as number,
+  tags_for_question (q.id) AS tags,
+  ag.number AS alternative_group_number,
+  ag.number_choose AS alternative_group_number_choose,
+  (
+    count(*) OVER (
+      PARTITION BY
+        ag.number
+    )::integer
+  ) AS alternative_group_size,
+  z.title AS zone_title,
+  z.number AS zone_number,
```

Comment:
A huge amount of this query is just about getting the issue count, you should be able to get rid of all that

Reply:
I removed the issue counts portion from the query and the page